AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/work-allocation.component.html

```diff
@@ -72,7 +72,8 @@ <h1 class="govuk-heading-l">Manage work allocation</h1>
           <ng-container *ngSwitchDefault>
             <p>
               <fa-icon icon="circle-exclamation" aria-hidden="true"></fa-icon>
-              Team {{ isWorkingHoursUpload ? 'working' : 'non-availability' }} hours upload partially successfully. Below user(s) could not be found:
+              Team {{ isWorkingHoursUpload ? 'working' : 'non-availability' }} hours upload partially successfully. Below user(s) could not
```

Comment:
I know you didn't write this message originally - any chance we could fix the typo? hours upload partially **successful**

Reply:
Thanks. No provlem. Will update it, as part of addressing the other comments